modules/hostap/src/supp_api.c

```diff
@@ -1408,6 +1410,75 @@ int hapd_config_network(struct hostapd_iface *iface,
 }
 #endif
 
+int supplicant_ap_status(const struct device *dev, struct wifi_iface_status *status)
+{
+	int ret = 0;
+	struct hostapd_iface *iface = NULL;
+	struct hostapd_config *conf = NULL;
+	struct hostapd_data *hapd = NULL;
+	struct hostapd_bss_config *bss = NULL;
```

Comment:
No need to set these to null because you are setting values for the variables below.

Reply:
modified